frontend/src/routes/manage/Realm/Content/AddButtons.tsx

```diff
@@ -125,7 +125,10 @@ export const AddButtons: React.FC<Props> = ({ index, realm }) => {
                     padding: 0,
                     "& > li": {
                         "&:not(:last-child)": {
-                            borderBottom: `1px solid ${isDark ? COLORS.grey4 : COLORS.grey2}`,
+                            borderBottom: `1px solid ${isDark
+                                ? COLORS.neutral35
+                                : COLORS.neutral20
+                            }`,
```

Comment:
Why split this in multiple lines?

Reply:
It's 102 characters otherwise. But the split doesn't make much sense, that's true :D
doing two lines like this works:
```suggestion
                          borderBottom:
                                `1px solid ${isDark ? COLORS.neutral35 : COLORS.neutral20}`,

Reply:
...ignoring the linebreak that github introduced of course

Reply:
Ah wow it's already 102. Then I don't care which of the two versions we use :shrug: